symphony-bdk-core/src/main/java/com/symphony/bdk/core/SymphonyBdk.java

```diff
@@ -105,7 +113,10 @@ protected SymphonyBdk(BdkConfig config, ApiClientFactory apiClientFactory)
    * @return {@link MessageService} message service instance.
    */
   public MessageService messages() {
-    return this.messageService;
+    if (this.messageService != null) {
```

Comment:
One could use `Optional.ofNullable(this.messageService).orElseThrow( () -> new NoBotConfigException(NO_BOT_CONFIG_MESSAGE))`. And we could put that in common into a dedicated method `private T getOrThrowNoBotConfig(T field)` or similar.